addons/point_of_sale/static/src/app/screens/scale_screen/scale_screen.js

```diff
@@ -94,4 +92,24 @@ export class ScaleScreen extends Component {
             this.state.tareLoading = false;
         }, 3000);
     }
+    get productPrice() {
+        const product = this.props.product;
+        const priceUnit = product.get_price(this._activePricelist, 1);
+        const taxes = product.taxes_id;
```

Comment:
What if there is a fiscal position on the order ? Check with Mohamed if you need to display the price before or after fiscal position. If you need to include the fiscal position here is my suggestion
```suggestion
        let taxes = product.taxes_id;
        if (order.fiscal_position_id) {
            taxes = getTaxesAfterFiscalPosition(taxes, this.pos.get_order().fiscal_position_id, this.pos.models);
        }
```